types/types.go

```diff
@@ -200,7 +200,7 @@ type receiptMarshaling struct {
 	TransactionIndex  hexutil.Uint64  `json:"transactionIndex"`
 	TxExecErrorMsg    *string         `json:"txExecErrorMsg"`
 	Type              *hexutil.Uint   `json:"type,omitempty"`
-	L1Fee			  hexutil.Uint64  `json:"l1Fee,omitempty"`
+	L1Fee			  *hexutil.Big  `json:"l1Fee,omitempty"`
```

Comment:
Looks good. Please test before merging. And please also fix the whitespace/alignment issue